src/components/layouts/DroidconOrganizers.tsx

```diff
@@ -4,130 +4,41 @@ import { layoutProperties } from "../../constants/Properties";
-			facebook: "droidconke",
-			twitter: "droidcon.co.ke",
-			instagram: "droidcon.co.ke",
-			logo: "https://secure.meetupstatic.com/photos/event/8/d/7/4/clean_489516212.jpeg",
-			slug: "droidcon-ke-645",
-			status: "active",
-			created_at: "2021-09-10 16:26:32",
-			upcoming_events_count: 1,
-			total_events_count: 1,
-		},
-		{
-			id: 4,
-			name: "app labs",
-			email: "droidconke@gmail.com",
-			description: "droidcon Ke",
-			facebook: "droidconke",
-			twitter: "droidcon.co.ke",
-			instagram: "droidcon.co.ke",
-			logo: "https://appslab.co.ke/assets/img/logo.png",
-			slug: "droidcon-ke-645",
-			status: "active",
-			created_at: "2021-09-10 16:26:32",
-			upcoming_events_count: 1,
-			total_events_count: 1,
-		},
-		{
-			id: 5,
-			name: "early camp",
-			email: "droidconke@gmail.com",
-			description: "droidcon Ke",
-			facebook: "droidconke",
-			twitter: "droidcon.co.ke",
-			instagram: "droidcon.co.ke",
-			logo: "https://media-exp1.licdn.com/dms/image/C4D0BAQFbXBKf5A1wxQ/company-logo_200_200/0/1533933103212?e=2147483647&v=beta&t=BIObWjP27BUIyzN5H9finiwLhwSlrlKc5fEcvTiki4s",
-			slug: "droidcon-ke-645",
-			status: "active",
-			created_at: "2021-09-10 16:26:32",
-			upcoming_events_count: 1,
-			total_events_count: 1,
-		},
-		{
-			id: 6,
-			name: "tiskos kenya",
-			email: "droidconke@gmail.com",
-			description: "droidcon Ke",
-			facebook: "droidconke",
-			twitter: "droidcon.co.ke",
-			instagram: "droidcon.co.ke",
-			logo: "https://pbs.twimg.com/profile_images/1297998197630087169/zvpGWoVq_400x400.jpg",
-			slug: "droidcon-ke-645",
-			status: "active",
-			created_at: "2021-09-10 16:26:32",
-			upcoming_events_count: 1,
-			total_events_count: 1,
-		},
-	],
-};
+const DroidconOrganizers = () => {
 
-interface OrganizersData {
-	id: number;
-	name: string;
-	email?: string;
-	description?: string;
-	facebook?: string;
-	twitter?: string;
-	instagram?: string;
-	logo: string;
-	slug?: string;
-	status?: string;
-	created_at?: string;
-	upcoming_events_count?: number;
-	total_events_count?: number;
-}
+	// Redux dispatch.
+	const dispatch = useAppDispatch();
 
-const DroidconOrganizers = () => {
-	const [organizersData, setOrganizersData] = useState<OrganizersData[]>([]);
+	const { sponsors } = useAppSelector(state => state);
+
+	const { data } = useAppSelector((state) => state.organizers);
+
+	const [trigger, { data: organizersData, error: getOrganizersError, isLoading: getOrganizersIsLoading, isSuccess: getOrganizersIsSuccess, isError: getOrganizersIsError}] = useLazyGetOrganizersQuery()
+
+	useEffect(() => {
+		if(sponsors?.data?.length !== 0) {
+			// Trigger fetch organizers query.
+			trigger();
+		}
+	},[sponsors])
 
-	const loadOrganizersData = async () => {
+	useEffect(() => {
 		//TODO: obtain data/response from the api service
 		//set data
-		setOrganizersData(MOCK_DATA.data);
-	};
+		console.log({ organizersData, getOrganizersError, getOrganizersIsLoading, getOrganizersIsSuccess, getOrganizersIsError })
```

Comment:
Hello @brianwachira 

I assume this won't be necessary in the final prod version?